<?xml version="1.0" encoding="utf-8"?>

<launch>

    <node
        name="search_yolo_node"
        pkg="yolov8_ros"
        type="yolov8_node.py"
        output="screen">
        
        <param name="config_file" value="search_node.yaml" />

    </node>

</launch>
